jackson-modules/jackson-conversions-2/src/test/java/com/baeldung/jackson/tocollection/DeserializeToJavaCollectionUnitTest.java

```diff
@@ -0,0 +1,130 @@
+package com.baeldung.jackson.tocollection;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.type.CollectionType;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import org.assertj.core.util.Lists;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class DeserializeToJavaCollectionUnitTest {
+    private ObjectMapper objectMapper;
+    private XmlMapper xmlMapper;
+    private List<Book> expectedBookList;
+
+
+    @BeforeEach
+    void setup() {
+        objectMapper = new ObjectMapper();
+        xmlMapper = new XmlMapper();
+        expectedBookList = Lists.newArrayList(
```

Comment:
two space indents (instead of four) on all of the 'new ...' lines

Reply:
fixed.